test/unit/pipeline/show-tabs.ts

```diff
@@ -1,17 +1,21 @@
 /*
-	File: src/test/unit/pipeline/show-tabs.ts
+	File: test/unit/pipeline/show-tabs.ts
 	cpuabuse.com
 */
 
-import { assert } from "assert";
-import {inputData} from "../../../test/input/pipeline.js"
-import {expected} from "../../../test/expected/pipeline.js"
-import {showTabs} from "../../../src/pipeline/show-tabs.js"
+/**
+ * Tests `showTabs`.
+ */
 
-function testShowTabs() {
-    describe("tab character", function(){
-        it("should have expected value", function(){
-            assert.strictEqual(showTabs.processTabs(inputData.tabCharacter), expected.tabCharacter)
-        });
-    });
-}
\ No newline at end of file
+import { strictEqual } from "assert";
+import { fruitTable as input } from "../../input/pipeline";

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "../../expected/pipeline"](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5042286)